tests/playbooks/tests_team_plugin_installation.yml

```diff
@@ -3,7 +3,11 @@
 - name: Play for testing team plugin installation
   hosts: all
   tasks:
-    - name: Check if rpm ostree system - cannot test
+    - name: >-
+        This is an rpm-ostree system.  This test cannot be run on such a system.
```

Comment:
There are some double spaces in the text.

Reply:
Wow, never heard of it. I am not against you can keep it if so. Otherwise, lgtm.